src/content/governance/index.md

```diff
@@ -0,0 +1,151 @@
+- **Miners/Validators**: these people run nodes that can add new blocks to the Ethereum blockchain;
+- **Node Operators**: these people run nodes that propagate blocks and transactions, rejecting any invalid transaction or block that they come across;
+- **Application/Tooling Developers**: these people write applications that are run on the Ethereum blockchain (e.g. DeFi, NFTs, etc.) or build tooling to interact with Ethereum (e.g. wallets, test suites, etc.);
+- **Application Users**: these people interact with applications on the Ethereum blockchain;
+- **Ether holders**: these people own the ETH token.
+
+_Note: any individual can be part of multiple of these groups (e.g. a protocol developer could champion an EIP, and run a beacon chain validator, and use DeFi applications). For conceptual clarity, it is easiest to distinguish between them, though._
+
+<Divider />
+
+## What is an EIP? {#what-is-an-eip}
+
+One important process used in Ethereum Governance is the proposal of **Ethereum Improvement Proposals (EIPs)**. EIPs are standards specifying potential new features or processes for Ethereum. Anyone within the Ethereum community can create an EIP. For example, none of the authors of EIP-721, the EIP that standardized NFTs, have worked directly on Ethereum's protocol development.
+
+<ButtonLink to="/eips/">More on EIPs</ButtonLink>
+
+<Divider />
+
+## The Formal Process {#formal-process}
+
+The formal process for introducing changes to the Ethereum protocol is as follows:
+
+1. **Propose a Core EIP**: as described in [EIP-1](https://eips.ethereum.org/EIPS/eip-1#core-eips), the first step to formally proposing a change to Ethereum is to detail it in a Core EIP. This will act as the official specification for an EIP that Protocol Developers will implement if accepted.
+
+2. **Present your EIP to Protocol Developers**: once you have a Core EIP, you should get feedback on it from Protocol Developers. You can get feedback can by proposing it for discussion on an [AllCoreDevs call](https://github.com/ethereum/execution-specs/tree/master/network-upgrades#getting-the-considered-for-inclusion-cfi-status). Alternatively, discussion can happen asynchronously on the [Ethereum Magician's forum](https://ethereum-magicians.org/) or in the [Ethereum R&D Discord](https://discord.gg/mncqtgVSVw).
+
+> Potential outcomes of this stage are:
+
+> - The EIP will be considered for a future network upgrade
+> - Technical changes will be requested
+> - It may be rejected if it is not a priority or the improvement is not large enough relative to the development effort.
+
+3. **[Optional] Gather Community Consensus for your EIP**: some EIPs are purely technical changes with little room for debate, but others can be more contentious. If the EIP is in the latter category, Protocol Developers will likely want to see strong evidence of overwhelming community support before proceeding with it. There is no "one size fits all" approach here, as each EIP is unique.
```

Comment:
Should this step happen before presenting to protocol devs? Seems like building some amount of community support for you idea should happen first to ensure an EIP is seriously considered.

Reply:
Not sure. @timbeiko any thoughts here?

Reply:
I would actually prefer removing this section entirely.  It is impossible to prove that you have "community consensus", and it is fairly easy to make it appear that someone else doesn't have "community consensus".  While advertising/advocating for an EIP is valuable in terms of attracting the right talent to work on it and/or making people aware of potential future changes, Ethereum's governance process is not a popularity contest and that is the only thing a step like this achieves.

Reply:
I agree @MicahZoltu but to your point above:

> We should be having people get feedback from the community prior to AllCoreDevs call.

Seems like we should have some step 2. before "Present your EIP to Protocol Developers" - any suggestion here? Perhaps we rephrase this step 3. to "Gather Community Feedback for your EIP" & move it before the ACD step?

Reply:
~~I proposed another bullet for community feedback pre-ACD calls. I'm not sure what to do of this section. For larger EIPs, there definitely is a sort of "community consensus" bit once ACD is somewhat in favor. 1559, 3074 and ProgPoW all arguable spent most of their time in this zone. How would you frame that, @MicahZoltu ?~~

I'm fine moving everything "community consensus" related to the section below. But, there should probably be another bullet that explains the technical back and forth that will happen on an EIP from the first presentation on AllCoreDevs to getting included in an upgrade. Something like:

> **Iterate towards a final proposal**: After receiving feedback from all relevant stakeholders, you will likely need to make changes to your initial proposal to improve its security, better meet the needs of various users, etc. Once your EIP has incorporated all the changes you believe are necessary, you will need to present it again to Protocol Developers. You will then either move to the next step of this process, or new concerns will be raised which will require another round of iterations on your proposal.